scripts/start.sh

```
@@ -0,0 +1,21 @@
+
+# Wait for PostgreSQL to be ready
+echo "Waiting for PostgreSQL to be ready..."
+until docker exec scribe-ai-postgres-container pg_isready -U user -d scribe.ai; do
+  echo "PostgreSQL is unavailable - sleeping"
+  sleep 1
+done
+
+echo "PostgreSQL is ready! Running migrations..."
+
+# Run Prisma migrations
+cd "$PROJECT_ROOT/src/api"
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)